static/js/reducers/channel_subscribers_test.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import { actions } from "../actions"
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { makeSubscriber } from "../factories/channels"
+
+describe("channelSubscribers reducers", () => {
+  let helper, dispatchThen, newSubscriber
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper()
+    newSubscriber = makeSubscriber("new_mod")
+    dispatchThen = helper.store.createDispatchThen(
+      state => state.channelSubscribers
+    )
+    helper.addChannelSubscriberStub.returns(Promise.resolve(newSubscriber))
+    helper.deleteChannelSubscriberStub.returns(Promise.resolve())
```

Comment:
Is this needed?

Reply:
Ah, I'll remove that

Reply:
Actually it is needed for the test for deleting a subscription